scripts/build_cdc_data_and_tag_latest.sh

```diff
@@ -14,36 +14,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Creates a new custom DC data docker image and tags it latest.
+# Also tags it with a custom label from an argument.
 
-# Creates a new custom DC data docker image and tags it as latest.
-
-# Usage: From root, ./scripts/build_cdc_data_and_tag_latest.sh
+# Usage: From root, ./scripts/build_cdc_data_and_tag_latest.sh $IMAGE_LABEL
 
 # The latest image = gcr.io/datcom-ci/datacommons-data:latest
 
 set -e
 set -x
 
-# Initialize all submodules, then merge them with their masters and update
-# their pinned versions (locally only).
-./scripts/update_git_submodules.sh
-./scripts/merge_git_submodules.sh
-
-# Configure Git to create commits with Cloud Build's service account
-git config user.email $(gcloud auth list --filter=status:ACTIVE --format='value(account)')
-
-git commit --allow-empty -am "DO NOT PUSH: Temp commit to update pinned submod versions (empty if submods are already up-to-date)"
-
-website_rev="$(git rev-parse --short HEAD)"
-# Even though data docker does not use mixer, we use it to generate the tag so that the tags are consistent between data and services docker images.
-mixer_rev="$(git rev-parse --short HEAD:mixer)"
-import_rev="$(git rev-parse --short HEAD:import)"
-image_label="${website_rev}-${mixer_rev}-${import_rev}"
+image_label=$1
+if [[ $image_label = "" ]]; then
+  echo "Expected positional argument with image label."
```

Comment:
nit: May be also `echo "Usage: ./scripts/build_cdc_data_and_tag_latest.sh $IMAGE_LABEL"` here and elsewhere?

Reply:
Done, here and in the service build script.